cdap-docs/developers-manual/source/building-blocks/spark-programs.rst

```diff
@@ -249,6 +249,24 @@ Spark in Workflows
 Spark programs in CDAP can also be added to a :ref:`workflow <workflows>`, similar to a :ref:`MapReduce <mapreduce>`.
 
 
+Spark SQL
+=========
+The entry point to functionality in Spark SQL is through a Spark `SQLContext
+<http://spark.apache.org/docs/latest/sql-programming-guide.html#starting-point-sqlcontext>`__.
+To run a Spark SQL program in CDAP, you can obtain a ``SQLContext`` from CDAP's ``SparkContext``
+using either of these methods:
```

Comment:
either of these sounds like they can use anyone among the below two but here they can only use one depending upon the language they are using. Do you think we can use some other word ? 

Besides the above comment LGTM.